tuist/tuist
# Support macOS apps as dependencies

Resolves https://github.com/tuist/tuist/issues/4973

### Short description 📝

- Enable adding macOS apps as dependencies for macOS (e.g. login items)

### How to test the changes locally 🧐

1. Checkout a sample project:
- https://github.com/serejahh/tuist/tree/fix/embed-apps-into-mac-apps/projects/tuist/fixtures/macos_app_with_app_inside
2. Run `tuist generate`
3. Make sure project is generated

### Contributor checklist ✅


### Reviewer checklist ✅
